chanzuckerberg/idseq-web
# [Report Page] Add expand all icon to expand all genus rows on the header

# Description

* Add expand all icon to expand all genus rows on the header
* Icon should also reflect the state when all the rows are expanded or collapsed even if they were expanded/collapsed individually.

# Tests

* Verify that expanding and collapsing from the header expands and collapses all rows
* Apply a few filters, expand all rows individually - verify that the header's expand/collapse icon goes to the collapsable position
